File: Python/Program/AnggotaTim.py
# File: AnggotaTim.py
from RiwayatTurnamen import RiwayatTurnamen

class AnggotaTim:
    def __init__(self, id_anggota="", nama="", nickname=""):
        self._id_anggota = id_anggota
        self._nama = nama
        self._nickname = nickname
        self._riwayat_turnamen = []
    
    def add_riwayat(self, riwayat):
        self._riwayat_turnamen.append(riwayat)
        
    def get_id_anggota(self): return self._id_anggota
    def get_nama(self): return self._nama
    def get_nickname(self): return self._nickname
    def get_riwayat_turnamen(self): return self._riwayat_turnamen

File: Java/Program/AnggotaTim.java
import java.util.ArrayList;

public class AnggotaTim {
    protected String idAnggota;
    protected String nama;
    protected String nickname;
    protected ArrayList<RiwayatTurnamen> riwayatTurnamen;

    public AnggotaTim(String id, String nama, String nickname) {
        this.idAnggota = id;
        this.nama = nama;
        this.nickname = nickname;
        this.riwayatTurnamen = new ArrayList<>();
    }

    public void addRiwayat(RiwayatTurnamen riwayat) {
        this.riwayatTurnamen.add(riwayat);
    }

    public String getIdAnggota() { return this.idAnggota; }
    public String getNama() { return this.nama; }
    public String getNickname() { return this.nickname; }
    public ArrayList<RiwayatTurnamen> getRiwayatTurnamen() { return this.riwayatTurnamen; }
}